src/modules/Tab/Tab.js

```diff
@@ -96,19 +102,42 @@ class Tab extends Component {
     })
   }
 
+  renderColumn(columnProps) {
+    return GridColumn.create(columnProps)
+  }
```

Comment:
Since this is a pass-through method, let's just call `GridColumn.create` instead of `this.renderColumn`.